content/docs/intro/languages/_index.md

```diff
@@ -16,6 +16,7 @@ capable as the other and supports the entire surface area of resources available
 
 The following language runtimes are currently supported by Pulumi:
 
+* [.NET]({{< relref "dotnet.md" >}}) <span class="badge badge-preview">PREVIEW</span> - C#, F# and Visual Basic
```

Comment:
Let's move this below Python and above Go.  Grouping preview items together and ordering in terms of current completeness.

Reply:
accepted